tailwind-annotator.config.mjs

```diff
@@ -4,7 +4,6 @@ export default {
   presets: [tailwindConfig],
   content: [
     './src/annotator/components/**/*.{js,ts,tsx}',
-    './src/shared/components/**/*.{js,ts,tsx}',
```

Comment:
We no longer have components in shared folder, but I'm not completely sure if we should keep this config in case we add some components in future.

Reply:
I would probably keep it, since it seems quite possible that we'll want this again in future.